bothub/api/v2/repository/views.py

```diff
@@ -7,10 +8,12 @@
 from drf_yasg.utils import swagger_auto_schema
 from rest_framework.decorators import action
 from rest_framework.exceptions import ValidationError
+from rest_framework.exceptions import UnsupportedMediaType
```

Comment:
When you import something, ensure that the new importation is in alphabetical order.

Reply:
When documentation is released to standardize projects, I will apply this change.